CRADLE/CallPeak/callPeak.py

```diff
@@ -476,19 +522,20 @@ def run(args):
 		outputCount = int(finalResult[i][8])
 		neglogPvalue = float(finalResult[i][4])
 		cohens_D = float(finalResult[i][9])
-		if(np.isnan(neglogPvalue) == True):
+		peusdoLog2FC = float(finalResult[i][10])
+		if np.isnan(neglogPvalue) == True:
```

Comment:
Thank you for removing the parenthesis!
```suggestion
		if np.isnan(neglogPvalue):
```